network/kademlia.go

```diff
@@ -833,7 +957,7 @@ func (k *Kademlia) knowNeighbours(addrs [][]byte) (got bool, n int, missing [][]
 	pm := make(map[string]bool)
 	depth := depthForPot(k.conns, k.NeighbourhoodSize, k.base)
 	// create a map with all peers at depth and deeper known in the kademlia
-	k.eachAddr(nil, 255, func(p *BzzAddr, po int) bool {
+	k.eachAddr(nil, nil, 255, func(p *BzzAddr, po int) bool {
```

Comment:
Why not `k.EachAddr(nil, 255, func(p *BzzAddr, po int) bool {`, and avoid that misterious `nil`?
Or even `k.eachAddr(nil, k.addrs, 255, func(p *BzzAddr, po int) bool {` seems more appropiate.